### DISCUSSION TITLE
Добавить API для загрузки звуков и изображений

### DISCUSSION THREAD

**Discussion by vitalets** [sberdevices/salute-issues#220] [SmartApp Code, SmartApp Graph, SmartApp API, Assistant Client]
Когда в приложении используется много звуков/изображений, то управлять этим вручную становится тяжело.
Было бы удобно иметь HTTP API, чтобы можно было автоматизировать это.
Аналогичное [API в Алисе](https://yandex.ru/dev/dialogs/alice/doc/resource-sounds-upload.html#http-load).

**Comment by Genenko-Svetlana** [comment]
@skobets
